frontends/concrete-python/concrete/fhe/compilation/module.py

```diff
@@ -184,10 +212,47 @@ def encrypt(
         assert isinstance(self.runtime, ExecutionRt)
         return self.runtime.client.encrypt(*args, function_name=self.name)
 
-    def run(
+    def run_sync(
```

Comment:
Let's keep this name as is. `run` and `run_async` is clear enough.

Reply:
What do you mean, you're changing `run` to `run_sync`, not introducing `run_sync` along `run` :thinking:

Reply:
maybe the test will make that clearer.
run do what the conf says
run_async and run_sync overrides the conf

Reply:
Ah you didn't remove run, but moved it below. The diff looked weird, sorry!